CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ endif()
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(GNUInstallDirs)
```

Comment:
What do we need this for?

Reply:
Portablitily variables. `CMAKE_INSTALL_DATADIR` and `CMAKE_INSTALL_INCLUDEDIR` are used in libvast_test.

Reply:
Good to know!